src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.Adapters.cs

```diff
@@ -12,6 +12,7 @@ public partial class SslStream
         private interface ISslIOAdapter
         {
             ValueTask<int> ReadAsync(byte[] buffer, int offset, int count);
+            ValueTask<int> ReadAsync(Memory<byte> buffer);
```

Comment:
Do we still need the `ValueTask<int> ReadAsync(byte[] buffer, int offset, int count);`  overload?

Reply:
It is used in FillBufferAsync() but that is easy to fix.